content/en/docs/best-practices/common.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Common recommendations and suggestions"
+linkTitle: "Common suggestions"
+weight: 2
+description: Common recommendations and suggestions to distribute solutions with Operator OLM
+---
+
+## Overview
+
+Any recommendation or good practice suggested by the Kubernetes community such as to develop [Operator pattern][operator-pattern] solutions or to manage them are good recommendations for who is looking for to build the operator projects and distribute them with OLM. Also, see [Operator Best Practices][operator-best-practices].
+
+## Validate your bundle before publish it
+
+Check and test your operator bundle before you publish it. Note that the [`operator-sdk`][operator-sdk] CLI can help with that process. You can validate a bundle via [`operator-sdk bundle validate`][sdk-cli-bundle-validate] against the entire suite of validators for Operator Framework, in addition to required bundle validators:
+
+```sh
+operator-sdk bundle validate ./bundle --select-optional suite=operatorframework
+```
+
+Also, you can validate a bundle via [`operator-sdk scorecard`][sdk-cli-scorecard-bundle] to insure it against a suite of tests:
+
+```sh
+operator-sdk scorecard bundle
+```
+
+## Provide what are the k8s versions supported by your project
+
+In this [CSV](/docs/concepts/crds/clusterserviceversion)  manifest, you will check that you can use the spec field `minKubeVersion` to inform what is the minimal Kubernetes version which your project supports, which would like: 
+
+```yaml
+  ...
+    spec:
+	  maturity: alpha
+	  version: 0.4.0
+	  minKubeVersion: 1.16.0
```

Comment:
This didn't affect the netlify prefix at all but this is being marked with red blocks when I look at these changes. Not too sure why, maybe extra spaces? Not too big of a deal as it's doesn't affect any actual content.